org/cloudfoundry.yml

```diff
@@ -5154,6 +5156,7 @@ orgs:
         - ccjaimes
         - gururajsh
         - PeteLevineA
+        - noxsingvm
```

Comment:
Note that this team will be going away with https://github.com/cloudfoundry/community/pull/387